src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static Task Timebox(this IEnumerable<Task> tasks, TimeSpan timeoutAfter,
 
                     if (t.IsFaulted && t.Exception != null)
                     {
-                        tcs.TrySetException(t.Exception.InnerException);
+                        tcs.TrySetException(t.Exception.GetBaseException());
```

Comment:
I'm not sure I understand the impact of this change. Could you explain this a little bit more? Was there an issue involved with this line?